### PR TITLE
Support missing e2ap failures

### PR DESCRIPTION
**Pull request by adibrastegarnia**

### CODE REVIEW

File: pkg/controller/v1beta1/subscription/controller.go
Comment (adibrastegarnia) [inline pkg/controller/v1beta1/subscription/controller.go:693]: Is this renaming correct? @eroshiva

**Review by kuujo — APPROVED** [review]

**Comment by gab-arrobo** [comment]
@adibrastegarnia, should this PR be closed?

**Comment by gab-arrobo** [comment]
> @adibrastegarnia, should this PR be closed?

@adibrastegarnia, given that this PR is already approved, should we plan to merge it? If so, please resolve conflict in `go.mod` and "update/delete" the `replace` directives you have there. Or are changes in `ono-api` and `onos-ric-sdk-go` needed before merging this PR?

**Pull request closed without merging**
